TodoApi/wwwroot/site.js

```diff
@@ -14,13 +15,22 @@ function getCount(data) {
 }
 
 $(document).ready(function () {
+    if (Cookies.get('CanAccess' === 'undefined')) {
```

Comment:
What is this Cookie? Doesn't appear that this is properly implemented or even used since we've got the header value instead